### PR TITLE
Reader: create a custom style for photo cards in Search

### PR DESCRIPTION
**Pull request by jancavan**
This PR creates custom styling for photo-only cards in Search to match the design:

screenshot 2016-07-08 16 23 33

Test live: https://calypso.live/?branch=update/reader/search-style-for-photo-cards

### CODE REVIEW

**Comment by blowery** [comment]
@jancavan we should add a test for this to docs/example.jsx...

**Comment by jancavan** [comment]
@blowery Good call. I'll add that in later. We probably need to add the Cold Start type cards on there as well.

File: client/components/post-card/search.jsx [hunk 1]
```diff
@@ -39,8 +39,10 @@ function SearchByline( { post, site, feed } ) {
 
 export function SearchPostCard( { post, site, feed, onClick = noop, onCommentClick = noop } ) {
 	const featuredImage = post.canonical_image;
+	const hasPost = !! post;
 	const classes = classnames( 'post-card__search', {
-		'has-thumbnail': !! featuredImage
+		'has-thumbnail': !! featuredImage,
+		'is-photo': hasPost && post.excerpt.length < 1
```
Comment (blowery) [inline client/components/post-card/search.jsx]: we have a DisplayType for PhotoOnly. should be able to say `&& ( post.display_type & DisplayTypes.PHOTO_ONLY )`

Comment (blowery) [inline client/components/post-card/search.jsx]: display types are in `import DisplayTypes from 'state/reader/posts/display-types';

**Comment by jancavan** [comment]
See http://calypso.localhost:3000/devdocs/app-components to test.

screenshot 2016-07-11 16 10 29

**Pull request merged**